src/app/components/landing-page/landing-page-dialog.component.ts

```diff
@@ -129,11 +128,12 @@ export class LandingPageDialogComponent implements OnInit, OnDestroy {
       () => {
         this.landingPageService.getServerCollections(url).then(
           () => {
-            this.urlCollectionsSubscription = this.configDescritor.getAllCollections().subscribe(
-              collections => {
-                this.availablesCollections = collections;
-                this.collectionService.setCollections(collections);
-                this.collectionService.getCollectionsReferenceDescription().subscribe(cdrs => this.collectionService.setCollectionsRef(cdrs));
+            this.urlCollectionsSubscription = this.collectionService.getCollectionsReferenceDescription().subscribe(
```

Comment:
For the other call to the method you added a pipe to takeUntil destroyed. Why not here ? Technically a pipe(take(1)) would work as well since once we have the response, we don't need the info again no ?

Reply:
Because we unsubscribe the subscription in the destroy of the component